custom/ibc-transfer/keeper/keeper.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	ibctransferkeeper "github.com/cosmos/ibc-go/v7/modules/apps/transfer/keeper"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	porttypes "github.com/cosmos/ibc-go/v7/modules/core/05-port/types"
+	"github.com/cosmos/ibc-go/v7/modules/core/exported"
+	ibctransfermiddleware "github.com/notional-labs/composable/v6/x/ibctransfermiddleware/keeper"
+)
+
+type Keeper struct {
+	ibctransferkeeper.Keeper
+	cdc                   codec.BinaryCodec
+	IbcTransfermiddleware *ibctransfermiddleware.Keeper
+	// authority         string
```

Comment:
why no authority?

Reply:
you need authority if you need to execute some code and check the sender of tx.
in our case there is no tx that would handle something.
so no need for some specific authority. 

the params is updatable via governance proposal that does not require any authority.
gov module will handle every update params by themself